packages/backend/src/helpers/backoff.ts

```diff
@@ -40,5 +40,11 @@ export const exponentialBackoffWithJitter: BackoffStrategy = function (
   const initialDelay = computeInitialDelay(err)
 
   const prevFullDelay = Math.pow(2, attemptsMade - 1) * initialDelay
-  return prevFullDelay + Math.round(Math.random() * prevFullDelay)
+  const totalDelay = prevFullDelay + Math.round(Math.random() * prevFullDelay)
+  logger.info('Job delay calculation', {
+    ...job?.data,
```

Comment:
nit: thinking we should enumerate this explicitly, might be a footgun in the future if we ever want to store sensitive data in `JobData`.

```suggestion
  flowId: job?.data?.flowId,
  executionId: job?.data?.executionId,
  stepId: job?.data?.stepId
```